app/models/location_image.rb

```diff
@@ -0,0 +1,8 @@
+class LocationImage < ApplicationRecord
+  belongs_to :location
+  has_one_attached :image
+
+  def sized_image
```

Comment:
you might want to allow the image size to be passed as a param.  See my example code.